providers/storetheindex/findproviders.go

```diff
@@ -0,0 +1,72 @@
+package storetheindex
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	logging "github.com/ipfs/go-log"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/multiformats/go-multihash"
+)
+
+var logger = logging.Logger("delegated/client")
```

Comment:
Was this intended to be merged with the other log, or not?

```suggestion
var logger = logging.Logger("sti/client")
```